app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,31 @@
+class SessionsController < ApplicationController
+  def new
+    redirect_to videos_path if logged_in?
+  end
+
+  def create
+    user = User.find_by(email: params[:email])
+
+    if user && user.authenticate(params[:password])
+      if(params[:remember_me]) == "1"
+        cookies.permanent.signed[:auth_token] = user.auth_token
+      else
+        cookies.signed[:auth_token] = user.auth_token
+      end
+
+      flash["notice"] = "You have been successfully logged in"
```

Comment:
Bootstrap 3 (which is used by this project) uses "success" and "danger" instead of "notice" and "alert".